source/administration/bulk-export-data.rst

```diff
@@ -0,0 +1,310 @@
+      <td valign="middle">string</td>
+      <td>The team description.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">allow_open_invite</td>
+      <td valign="middle">bool</td>
+      <td>Whether to allow open invitations. Will have one of the following values:<br>
+        <kbd>true</kbd><br>
+        <kbd>false</kbd>
+      </td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">scheme</td>
+      <td valign="middle">string</td>
+      <td>The name of the permissions scheme that applies to this team.</td>
+    </tr>
+  </table>
+
+Channel object
+--------------
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">team</td>
+      <td valign="middle">string</td>
+      <td>The name of the team this channel belongs to.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">name</td>
+      <td valign="middle">string</td>
+      <td>The name of the channel.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">display_name</td>
+      <td valign="middle">string</td>
+      <td>The display name for the channel.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">type</td>
+      <td valign="middle">string</td>
+      <td>The type of channel. Will have one the following values:<br>
+          <kbd>"O"</kbd> for a public channel.<br>
+          <kbd>"P"</kbd> for a private channel.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">header</td>
+      <td valign="middle">string</td>
+      <td>The channel header.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">purpose</td>
+      <td valign="middle">string</td>
+      <td>The channel purpose.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">scheme</td>
+      <td valign="middle">string</td>
+      <td>The name of the Scheme that applies to this team.</td>
+    </tr>
+  </table>
+  
+User object
+-----------
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">username</td>
+      <td valign="middle">string</td>
+      <td>The user’s username. This is the unique identifier for the user.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">email</td>
+      <td valign="middle">string</td>
+      <td>The user’s email address.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">auth_service</td>
+      <td valign="middle">string</td>
+      <td>The authentication service used for this user account. <br>
+        <kbd>""</kbd><br>
```

Comment:
OK so I'm actually quite confused about this one now. Will need to go check how the code actually runs.

Reply:
OK, so this line should be removed entirely, and in the previous line it should also say:

```
This field will be absent for user/password authentication.